packages/oauth/src/state-stores/interface.ts

```diff
@@ -1,18 +1,40 @@
 import { InstallURLOptions } from '../install-url-options';
 
-// State object structure
+/**
+ * The data structure represented by the state parameter.
+ */
 export interface StateObj {
+
+  /**
+   * The timestamp that the state object was generated.
+   */
   now: Date;
```

Comment:
Wanted to change this property name but we cannot do so for backward-compatibility